### DISCUSSION TITLE
Multitrack doesn`t work with peaks

### DISCUSSION THREAD

**Discussion by rassokhina** [katspaugh/wavesurfer.js#2918] [Q&A] [answered]
It seams that multitrack doesn't work with peaks. I'm trying to uncomment peaks in the example https://wavesurfer-js.org/examples/#multitrack.js, and they are not being rendered. Maybe I'm doing something wrong.. Thank you!

**Comment by katspaugh** [comment, marked as answer]
Good catch! It's been fixed in beta.10 but I haven't updated the dependency in multitrack. It should be good now. See https://github.com/katspaugh/wavesurfer-multitrack/commit/91de8d0881f26b91f58cbdbf7f9ef9f6a1651837

**Reply by rassokhina** [reply]
Example is working now, thank you @katspaugh
